### PR TITLE
Release 2023.04.0

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Adding release notes and docs for 2023.04.0
- **Is this PR related to a Monolith release?** Yes. 2023.04.0

### Resolves:

* [OKTA-595640](https://oktainc.atlassian.net/browse/OKTA-595640)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md [hunk 1]
```diff
@@ -2,6 +2,64 @@
...
+
+#### Configurable rate limits for OAuth 2.0 apps is GA in Production
+
+Rate limit violations mainly occur on authenticated endpoints. Currently, it isn't clear which OAuth 2.0 authenticated app consumes all the rate limits for an org. This increases the risk that one app consumes the entire rate limit bucket. To avoid this possibility, Okta admins can now configure how much rate limit capacity an individual OAuth 2.0 app can consume by editing the Application rate limits tab for each app. By setting a capacity on individual OAuth 2.0 apps, Okta admins have a new tool to monitor and investigate rate limit violations, and have the ability to view rate limit traffic generated by individual OAuth 2.0 apps. See [Rate limit dashboard bar graph](/docs/reference/rl-dashboard/#bar-graph). <!--OKTA-573387-->
+
+#### Bug fixed in 2023.04.0
+
+The Identity Sources API bulk upsert operation accepted an empty profile payload. (OKTA-533011)
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023/index.md:61]: FYI ... I noticed that this bug was also tagged for OIE RNs, but doesn't appear. Is this an error on the ticket? Or should it be in OIE?

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 2]
```diff
@@ -4,6 +4,66 @@ title: Okta Identity Engine API Products release notes 2023
...
+
+#### OAuth 2.0 grant scopes added
+
+The OAuth 2.0 `okta.appGrants.manage` and `okta.appGrants.read` grant scopes are now available for use with the `/apps/{id}/grants` and `/apps/{id}/grants/{grantId}` endpoints.<!--OKTA-568970-->
+
+#### Scope parameter length increased
+
+The maximum length for the scope parameter of a refresh token request is now 4096 characters.<!--OKTA-573127-->
+
+#### OAuth 2.0 On-Behalf-Of Token Exchange is GA in Preview
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:59]: When I clicked through to this page, there was still an EA tag. Was there a PR added to remove?

**Review by brianduffield-okta — APPROVED** [review]
Nice job, Brent. Very clean. Only highlighted two issues that should be checked.

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md (hunk 1 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023/index.md:61]: Thanks @brianduffield-okta I followed the edited blurb spreadsheet. I see the OIE tag on the ticket. Will fix!

Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023/index.md:61]: Done.

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md (hunk 2 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:60]: @brianduffield-okta Nope. No PR. We had a "remove EA tag" PR for another feature. I'll follow up with Susan. Thanks!

**Pull request merged**